src/contractTest/java/uk/gov/hmcts/reform/em/npa/consumer/NpaPactConsumerTest.java

```diff
@@ -272,4 +272,37 @@ void testDeleteMarkupsByDocumentId(MockServer mockServer) {
                 .then()
                 .statusCode(HttpStatus.OK.value());
     }
+
+    @Pact(consumer = "em_npa_api", provider = "native_pdf_annotator_api_provider")
+    public V4Pact deleteSingleMarkupPact(PactBuilder builder) {
+        return builder
+                .usingLegacyDsl()
+                .given("A valid RedactionDTO with ID exists")
```

Comment:
we should keep provide and consumer state the same I.e. "A valid Redaction with document ID and redaction ID exists and can be delete"